### DISCUSSION TITLE
✈ 📚Organize the sidebar Table of Contents in Ascending Alphabetical Order in the Developers Documentation (Migrated from Issue #28220)

### DISCUSSION THREAD

**Discussion by m1ga** [elementor/elementor#30548] [Feature Request]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

It would be a lot easier/quicker to browse the control documentation at: https://developers.elementor.com/docs/editor-controls/ if the names are ordered alphabetically:

![Bildschirmfoto_20240726_203644](https://github.com/user-attachments/assets/5edadfe8-8646-4b0c-b8ff-280c7aa9b2e5)

same for all sub menus in that section on the left side.

### Steps to reproduce

open https://developers.elementor.com/docs/editor-controls/ and try to find the "font control" without using the search :smile: 

### Expected behavior

sort the menu in alphabetical order 

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [ ] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

**[Redacted by Moderation]**

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hey @m1ga!

Thank you for sharing your feedback! 🙂 

We will forward this to the team responsible for the Developers Documentation, and if necessary we will make the appropriate changes! 💪 

Best regards! 👊
